c/tskit/tables.c

```diff
@@ -4954,13 +4954,132 @@ tsk_table_sorter_sort_mutations_canonical(tsk_table_sorter_t *self)
     return ret;
 }
 
+static int
+tsk_table_sorter_sort_individuals(tsk_table_sorter_t *self)
+{
+    int ret = 0;
+    tsk_id_t i;
+    tsk_individual_table_t copy;
+    tsk_individual_t individual;
+    tsk_individual_t copy_individual;
```

Comment:
you don't use `individual` and `copy_individual` at the same time, so maybe they should both be `individual`?